src/utils/splitHandlerPathAndName.js

```diff
@@ -1,6 +1,10 @@
 // some-folder/src.index => some-folder/src
 export default function splitHandlerPathAndName(handler) {
   // Split handler into method name and path i.e. handler.run
+  const prepathDelimiter = handler.lastIndexOf('/')
```

Comment:
Wouldn't it be easier to do a split on `/` here?

Reply:
I played around with this. Doing a split would break the the ruby logic and would also some not too easy to understand flows for determining the handler `path` , `name` and `moduleNestings`.

Let me know what you think. thanks